CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ option(ENABLE_COVERAGE "Enable coverage (only supported with GCC)" Off)
 option(ENABLE_FORCEINLINE_RECURSIVE "Enable Forceinline recursive (only supported with Intel compilers)" On)
 option(ENABLE_BENCHMARKS "Build benchmarks" Off)
 option(RAJA_DEPRECATED_TESTS "Test deprecated features" Off)
+option(ENABLE_BOUNDS_CHECK "Enable bounds checking in RAJA::Views" Off)
```

Comment:
I think we should  be explicit about which CMake variables are specific to RAJA and which are not; e.g., include a prefix such as "RAJA_" on those that our ours. So, I would prefer this to be RAJA_ENABLE_BOUNDS_CHECK or similar. Also, we should be consistent in the names of these variables; e.g., RAJA_DEPRECATED_TESTS should be RAJA_ENABLE_DEPRECATED_TESTS. @artv3 @davidbeckingsale @MrBurmark @ajkunen what do you think?

Reply:
I agree

Reply:
@artv3 let's address the CMake variable name change on a different PR. I'll discuss with you.

Reply:
I like that idea, I can change that in this PR.